src/3d.js

```diff
@@ -123,6 +223,84 @@ export var hackOptions = {
 	// but if you have a big, highly branching game with lots of art,
 	// you may want to disable it
 	preloadTextures: true,
+
+    // function used to adjust mesh instances after they have been added to the scene
+	meshExtraSetup: function (drawing, mesh) {
```

Comment:
ah i see... applyBehaviours sounds like just the right place to insert this kind of logic.. maybe it would make sense to include it in the hackoptions?

Reply:
oh right that was silly on my part... can just call meshExtraConfig from applyBehaviours